### ISSUE TITLE
Broken provided_by filter test

### ISSUE THREAD

**Issue by amykglen** [RTXteam/RTX#1549]
decided to create a separate issue for the broken filter test mentioned in https://github.com/RTXteam/RTX/issues/1519#issuecomment-871832904

`test_ARAX_filter_kg.py::test_provided_by_filter`

I wonder if that filter action should even bother checking the `attribute_type_id`? it might be hard for a user to know whether a given source is recorded as an `aggregator_knowledge_source` vs. just `knowledge_source`, or another type... maybe just look for an attribute whose `value` is equal to the infores curie? (e.g., `infores:semmeddb`)

**Comment by edeutsch** [comment]
This seems like an fine approach to me. Perhaps more rigorous to have a dictionary of the "source" attribute_type_ids check to make sure the supplied attribute_type_id is *in* that dict. but I can't think of a reason where the value would be infores:semmeddb and the attribute_type_id would be something else.

**Comment by finnagin** [comment]
Is there a general way to tell if it is a provided_by type attribute that would catch both aggregator_knowledge_source and knowledge_source attributes?

**Comment by finnagin** [comment]
I don't like the idea of just checking the value of the attributes because what if there are two different attributes that could have the same value but mean two different things?

**Comment by edeutsch** [comment]
In theory these are in BioLink, but maybe not until 2.0, I'm not sure:
```
biolink:knowledge_source (doesn't commit to the resource being aggregator, primary, or original. an abstract grouping property).
biolink:primary_knowledge_source (used for the furthest upstream resource in the chain that the data creator can identify)
biolink:original_knowledge_source (used when KP is confident that the primary source is the original source)
biolink:aggregator_knowledge_source (used when the KP is confident the resource is an aggregator)
biolink:supporting_data_source (used for resources providing data that a KP reasons with or computes on to generate new knowledge)
```
Would easier to have your own dict for now I think.
